### PR TITLE
Separate error prefix message and error message for different formater

### PR DESCRIPTION
**Pull request by unkcpz**
This PR contains two changes:
1. ~~The duplicate item will not add to the errors/warnings list of plugin from installation tests.~~ -> #288 
2. I added support for converting html string to react in showing the plugin checks warning and error messages. The install error message now has a head that shows the cause of the problem and the traceback for the exception of installation and import errors. This is also required by https://github.com/aiidateam/aiida-registry/pull/284 where I want to have a hyperlink to the README to guide the plugin maintainer on how to fix problem.

### CODE REVIEW

**Comment by AhmedBasem20** [comment]
Thanks @unkcpz, not sure what is the reason for the error duplication, I tried to have a look at the generated JSON file for this PR preview in `gh-pages` but it is not there. Also at the root `gh-pages` branch, the last version of https://github.com/aiidateam/aiida-registry/blob/gh-pages/plugins_metadata.json was pushed last week. Is this intended or maybe another issue? Anyway, this is not related to the duplication problem but I think it is good to bring `plugins_metadata.json` back to `gh-pages`.

**Comment by unkcpz** [comment]
> Also at the root gh-pages branch, the last version of https://github.com/aiidateam/aiida-registry/blob/gh-pages/plugins_metadata.json was pushed last week. Is this intended or maybe another issue? 

I also notice this is quite strange. But I think it is not the cause of the issue. I clean up the cache and it seems all works well now.

**Review by AhmedBasem20 — CHANGES_REQUESTED** [review]

**Comment by AhmedBasem20** [comment]
> I also notice this is quite strange. But I think it is not the cause of the issue. I clean up the cache and it seems all works well now.

Yeah, normally we should see the JSON file for every preview build. so it should be here for example https://github.com/aiidateam/aiida-registry/tree/gh-pages/pr-preview/pr-287

But this is not related to this PR we may open an issue and check this later.

File: aiida_registry/__init__.py [hunk 1]
```diff
@@ -165,31 +165,33 @@ def warn(self, string):
 
         Used to display log in actions.
         """
-        message = f"  > WARNING! {string}"
         # Set the step output error message which can be used,
         # e.g., for display as part of an issue comment.
+        output = f" > <strong>WARNING!</strong> {string}"
```
[the remark this answers is not in the document]
Comment (unkcpz) [reply aiida_registry/__init__.py]: Thanks. Done.

**Comment by unkcpz** [comment]
> Yeah, normally we should see the JSON file for every preview build. so it should be here for example https://github.com/aiidateam/aiida-registry/tree/gh-pages/pr-preview/pr-287

Yes, I noticed it as well but didn't manage to bring the file into `gh-pages` branch. Let's have a chat to see why I also have other questions in terms of test web build locally.

**Pull request merged**